xmodule/capa/correctmap.py

```diff
@@ -91,7 +91,7 @@ def set_dict(self, correct_map):
 
         """
         # empty current dict
-        self.__init__()
+        self.__init__()  # pylint: disable=unnecessary-dunder-call
```

Comment:
Changing the `__init__()` call with Class declaration and assigning it to `self` as `self=CorrectMap()` causes `self-cls-assignment` warning to be generated. 
Couldn't figure out what should be the way to go about this scenario.